prost-derive/src/lib.rs

```diff
@@ -23,6 +23,20 @@ fn try_message(input: TokenStream) -> Result<TokenStream, Error> {
 
     let ident = input.ident;
 
+    let recursion_limit: u32 = if let Some(attr) = input
+        .attrs
+        .iter()
+        .find(|attr| attr.path.is_ident("RecursionLimit"))
```

Comment:
I wonder if we should make this one camel case like serde does https://serde.rs/container-attrs.html

so `#[prost(recursion_limit = 5)]` etc

Reply:
I'm having trouble getting a bespoke attribute parser working. Would it be okay to pull in darling as a dependency? https://crates.io/crates/darling

Another option is to skip proper attribute parsing and only handle the single attribute we have for now.